requirements.txt

```diff
@@ -1,5 +1,5 @@
 requests>=2.4.0
 websocket-client>=0.22.0,<=0.44.0
-slacker>=0.9.50
+slack-sdk>=3.13.0
```

Comment:
update https://github.com/scrapinghub/slackbot/blob/develop/setup.py#L9 as well

Reply:
good call - this has now been updated